posts/verify-sync-release-volunteers/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Call for volunteers: help to test us the release syncing using staging instance"
+date: 2023-09-05T14:46:18+02:00
+authors: lbarczio
+tags:
+  - downstream
+  - pull-from-upstream
+  - propose-downstream
+  - staging
+---
+
+In the upcoming months, we plan to migrate our service to a new cluster. However, this may affect `propose_downstream`
+and `pull_from_upstream` jobs due to the new firewall rules. The problematic aspects could be:
+
+- commands you run in your `actions` during syncing the release involving interactions with external servers
+- downloading your sources from various hosting services (crates.io, npm, gems, etc.)
+
+To smoothen this transition, we kindly encourage you to enable one of these jobs on our already migrated staging instance.
+This recommendation is particularly important if you belong to one of the groups affected by the two previous points.
+This proactive step will help us identify and address any issues promptly.
+
+Both instances can be run at the same time and the behaviour can be configured via the `packit_instances` configuration key,
+which is by default set to `["prod"]`. Picking just one instance might be better for `koji_build` and `bodhi_update` jobs since both instances will work with the production instances of Fedora systems.  
```

Comment:
```suggestion
which is by default set to `["prod"]`. Picking just one instance is required only for `koji_build` and `bodhi_update` jobs since both instances work with the production instances of Fedora systems.  
```